index.html

```diff
@@ -12,6 +12,22 @@
     <!-- Header-section starts here -->
     <div class="hero-section">
       <header>
+
+        <!---------------dropdown-menu------------->
+        <div class='mobile-menu' id='mobile-menu'>
+              <div class="menu-x ">
+          <img src="./images/XIcon.svg" alt="close-menu"
+            onclick=closeMenu()  id="closeMenu"/>
+        </div>
+
+        <ul class='drop-menu-list'>
+            <li><a href="#work" aria-label ="portfolio area" class='target'>Portfolio</a></li>
+            <li><a href="#about" aria-label ="about me area">About</a></li>
+            <li><a href="#contact" aria-label ="contact me area">Contact</a></li>
+          </ul>       
+        </div>
+        <!---------------dropdown-menu------------->
```

Comment:
- [ ] **[OPTIONAL]:** Nice job working on your mobile menu! 💯 👏🏼 I noticed that you created different HTML for the menu. I would suggest you use the same markup from your desktop version and style it with CSS depending on the screen size (using media queries). If you double-check, both `<ul>` have exactly the same content. Please, consider working on this! 🤖 🤓

Reply:
Thank you, that is a good observation - will implement it onwards.